debugroute.py

```diff
@@ -32,11 +32,12 @@ class debugVertexRoute(baseVertex):
     vertex_type = 'route'
     non_config_obj = True
     def __init__(self, prefix=None, vn_fqname=None, ri_fqname=None,
-                 vrouters=None, **kwargs):
+                 vrouters=None, rt_type=None, **kwargs):
         self.vrouters = vrouters
         self.ri_fqname = ri_fqname
         self.vn_fqname = vn_fqname
         self.prefix = prefix
+        self.rt_type = rt_type
```

Comment:
Can we determine the route type if not provided?
You may refer https://github.com/Juniper/contrail-test-ci/blob/master/tcutils/util.py#L586